### DISCUSSION TITLE
Welcome to fwupd Discussions!

### DISCUSSION THREAD

**Discussion by superm1** [fwupd/fwupd#2659] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by YehezkelShB** [comment]
Don't you want to write the first comment yourself? :)

My name is Yehezkel Bernat. I contributed part of the Thunderbolt plugin as part of the work I did on Thunderbolt in Intel. I left Intel ~3 years ago, but I keep following the project to keep being involved when Thunderbolt is discussed and to learn from the way the project is handled and evolved.

**Comment by superm1** [comment]
You're too fast!  I didn't get a chance to make introduction as I got distracted.

I'm Mario Limonciello.  I have been involved since near the beginning and focus specifically on plugins related to Dell's client portfolio.  I am happy to provide guidance though and code review on other plugins.

**Comment by hughsie** [comment]
Hi all, I'm Richard Hughes, a.k.a. the grumpier of the two maintainers. I created fwupd as an offshoot of the ColorHug project and it somehow morphed into the open source component used by millions of people that we have now. I work for Red Hat and live in London with my little family. I'm happy to provide help with all aspects of the LVFS and fwupd.
